src/redux/action/action.js

```diff
@@ -5,6 +5,8 @@ import {
   ADDUSER_REQUEST,
   ADDUSER_SUCCESS,
   ADDUSER_FAILURE,
+  GET_USER,
+  GETTING_USER,
```

Comment:
rename GETTING_USER to USERS_RECEIVED everywhere, in function name also
GETTING_USER doesn't make sense